test/e2e/Makefile

```diff
@@ -64,6 +64,7 @@ CLOUD_PROVIDER ?=
 OBJECT_STORE_PROVIDER ?=
 INSTALL_VELERO ?= true
 REGISTRY_CREDENTIAL_FILE ?=
+GINKGO_SKIP ?=
```

Comment:
How about move it to the next line of #48

Reply:
OK.